shimming-toolbox/shimmingtoolbox/cli/b0shim.py

```diff
@@ -470,33 +482,39 @@ def _save_to_text_file_static(coil, coefs, list_slices, path_output, o_format, o
 
             list_fname_output.append(os.path.abspath(fname_output))
     else:  # o_format == 'gradient':
+        f0 = 'f0'
+        gradients = ['x', 'y', 'z']
+        if n_channels == 3:
+            name = {0: gradients[0],
+                    1: gradients[1],
+                    2: gradients[2]}
+        elif n_channels == 4:
+            name = {0: f0,
+                    1: gradients[0],
+                    2: gradients[1],
+                    3: gradients[2]}
```

Comment:
Could be a dictionary from the get-go as explained above

Reply:
Changed in [48e17b5](https://github.com/shimming-toolbox/shimming-toolbox/pull/560/commits/48e17b58778e642d35d2201835c259708abc99e9). This is a bit awkward though since some calls to save_txt_file has coefs in a dict, and some in a numpy array. We should try to move towards dicts in the future